schemas/configuration/v5/server_config.json.in

```diff
@@ -171,13 +304,34 @@
                     "minItems": 1
                 }
             },
-            "required": ["authentication","network","resource","rule_engines"]
+            "required": [
+                "authentication",
+                "network",
+                "resource",
+                "rule_engines"
+            ]
+        },
+        "plugin_directory": {
+            "type": "string",
+            "maxLength": 1087
         },
         "schema_name": {"type": "string"},
         "schema_version": {"type": "string"},
         "server_port_range_end": {"type": "integer"},
         "server_port_range_start": {"type": "integer"},
-        "tls": {
+        "tcp_keepalive_intvl_in_seconds": {
```

Comment:
can we use all the letters here?   interval?    unless this is coming from a spec somewhere else...

Reply:
oh, i see, it's a linux thing already, ... resolving.